multiuser_service_windows.go

```diff
@@ -0,0 +1,299 @@
+// +build multiuser
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
+	"golang.org/x/sys/windows/svc/mgr"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+// stderr handle is invalid when run as service
+var logWriter = io.MultiWriter(ioutil.Discard)
+
+func init() {
+	// TODO make this work with --working-directory
```

Comment:
I think that is done now, isn't it?

Reply:
This always changes the working directiry to the parent directory of the generic worker binary.